src/SignalR/common/Protocols.Json/src/baseline.netcore.json

```diff
@@ -0,0 +1,253 @@
+{
```

Comment:
We're still using these files even when we have reference assemblies?

Reply:
We purposefully removed all of these files from the master branch a while ago.